accepted/0000-caasp-nodes.md

```diff
@@ -0,0 +1,370 @@
+
+Any node of a CaaS Platform cluster can be registered to Uyuni/SUSE Manager.
+Registering a CaaS Platform node to Uyuni/SUSE Manager brings the following notable advantages:
+
+- The user can see the patch level status of the CaaS Platform nodes (at the packages level, not at the container level - the latter is called "[container staging](https://confluence.suse.com/display/~dmacvicar/Container+Staging+Breakout)" and it is outside of the scope of this RFC)
+- The user can perform configuration management operations using Salt from Uyuni/SUSE Manager
+- The user can assign a different set of channels (with channel staging and/or CLM filters) to different clusters and manually control from Uyuni/SUSE Manager which version of CaaS Platform gets automatically installed in its CaaSP clusters<sup>1</sup>.
+
+However, if the user does some specific operations on any node of the cluster via Uyuni/SUSE Manager or via plain Salt, the cluster may result temporarily or permanently unusable or broken.
+The whole idea behind this RFC is not to educate the final user in dealing with a CaaS Platform cluster, but rather to prevent any breakage in the operating cluster by issuing operations considered safe for usual systems.
+
+NOTE: the term "operation" in this RFC refers to any Salt command issued on a minion or any action issued on a system by Uyuni/SUSE Manager.
+
+This RFC describes how Uyuni/SUSE Manager can prevent the user to perform these kinds of actions on the CaaS Platform nodes.
+
+# Detailed design
+[design]: #detailed-design
+
+<!-- This is the bulk of the RFC. Explain the design in enough detail for somebody familiar with the product to understand, and for somebody familiar with the internals to implement.
+
+This section should cover architecture aspects and the rationale behind disruptive technical decisions (when applicable), as well as corner-cases and warnings. Whenever the new feature creates new user interactions, this section should include examples of how the feature will be used. -->
+
+## Terminology
+
+In this RFC, we will use:
+
+* CaaS Platform node: any node that comprises the cluster, either a control plane node or a worker node
+* Bootstrap/registration in the Uyuni/SUSE Manager context: registering a node to Uyuni/SUSE Manager via [the documented methods<sup>4</sup>](https://opensource.suse.com/doc-susemanager/suse-manager/client-configuration/registration-overview.html).
+At the time of writing, registering a CaaS Platform node to Uyuni/SUSE Manager has to be manually performed by the user.
+
+## Assumptions
+
+We assume that:
+* The user has already created an activation key and associated it with the onboarding. The CaaS Platform nodes are registered correctly and with the proper CaaS Platform channel(s) assigned. The
+activation key is needed to have `skuba-update` automatically patch the cluster with the latest patches available<sup>1</sup>.
```

Comment:
I would say it's need to allow CaaSP nodes to consume repositories managed by SUMA. As a nice "side-effect" `skuba-update` will also apply the patches published by SUMA.